crowdin.yml

```diff
@@ -1,13 +1,14 @@
 project_id_env: CROWDIN_PROJECT_ID
```

Comment:
Changes in this file are not required. Also typically there is no need to quote yaml files keys in quotes if they do not contain any special chars.

Reply:
Hmm, haven't touched this one. Must be a commit hook :/